src/components/Attachments/AttachmentCarousel/index.native.tsx

```diff
@@ -66,13 +68,14 @@ function AttachmentCarousel({report, reportActions, parentReportActions, source,
             Keyboard.dismiss();
             setShouldShowArrows(true);
 
-            const item = attachments[newPageIndex];
+            const item = attachments.at(newPageIndex);
```

Comment:
Same, please make sure it doesn't break anything in case `newPageIndex` is `-1`, cause .at() accepts negative integers while [] not. See https://github.com/Expensify/App/pull/49534#discussion_r1772650621